cpp/cmake/thirdparty/get_cucollections.cmake

```diff
@@ -12,14 +12,24 @@
 # the License.
 # =============================================================================
 
-# This function finds cuCollections and performs any additional configuration.
+# This function finds cucollections and sets any additional necessary environment variables.
 function(find_and_configure_cucollections)
-  include(${rapids-cmake-dir}/cpm/cuco.cmake)
-  if(BUILD_SHARED_LIBS)
-    rapids_cpm_cuco(BUILD_EXPORT_SET cudf-exports)
-  else()
-    rapids_cpm_cuco(BUILD_EXPORT_SET cudf-exports INSTALL_EXPORT_SET cudf-exports)
+
+  # Find or install cuCollections
+  rapids_cpm_find(
```

Comment:
Should we revert these changes for now and reconsider this switch when we can benchmark `pair_contains_if` with more complex types? @PointKernel what do you think?

Reply:
The problem is not necessarily with `pair_contains_if`. `pair_contains` (https://github.com/NVIDIA/cuCollections/pull/175) also requires a new git tag. I think we should directly update the cuco tag in `rapids-cmake` once 175 is merged.

Reply:
Oh I thought `pair_contains_if` was the only new API getting added. In that case, never mind :) Yes we need to update rapids-cmake.